package-lock.json

```diff
@@ -1,1237 +1,8 @@
 {
   "name": "vscode-stl-viewer",
   "version": "0.2.0",
-  "lockfileVersion": 2,
```

Comment:
Can you revert your change on this file?

Reply:
Yes you are right. Sorry.